scripts/build.py

```diff
@@ -110,21 +110,38 @@
 #### Telegraf Functions
 ################
 
+def print_banner():
```

Comment:
Did something go wrong with the banner formatting? I'm having trouble reading this....